holmes/plugins/toolsets/azuremonitor_metrics/azuremonitor_metrics_toolset.py

```
@@ -0,0 +1,1306 @@
+                            "description": essentials.get("description", ""),
+                            "severity": essentials.get("severity", ""),
+                            "alert_state": essentials.get("alertState", ""),
+                            "monitor_condition": essentials.get("monitorCondition", ""),
+                            "fired_time": essentials.get("firedDateTime", ""),
+                            "target_resource": essentials.get("targetResource", ""),
+                            "rule_details": rule_details
+                        }
+                        prometheus_alerts.append(alert_info)
+                
+                # Format the results nicely
+                result_data = {
```

Ruff (0.12.2)

Local variable `result_data` is assigned to but never used

Remove assignment to unused variable `result_data`

(F841)